pkg/snmp/traps/formatter.go

```diff
@@ -258,7 +258,8 @@ func enrichBits(variable trapVariable, varMetadata VariableMetadata) interface{}
 		}
 	}
 
-	return enabledValues
+	hexString := fmt.Sprintf("0x%x", bytes)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Pretty minor question but have we thought about lowercase vs uppercase for an hex string here? I usually tend to prefer uppercase as it's more explicit but that's a matter of personal preference

Reply:
Do we expose any other hex strings from SNMP as metric tags? We could try to match the casing of those... though it's also quite possible that it varies depending on the device.

Reply:
Ah good call, indeed if we were to match correlate this to a metric we may want the case to match to make it easier but I'm not sure we have any example right now, nor that it would make too much sense to me in the future unless such a bit string is used as an index of some table?